README.rst

```diff
@@ -40,8 +40,9 @@ respecting the standard procedures in each system:
 * Use standard configuration space in ``/etc`` on traditional production servers
 * Handle environment-based configuration for cloud-based platforms
 
-``getconf`` v1.6 onwards supports 2.7, 3.3, 3.4, 3.5, 3.6 and is distributed under the two-clause BSD license.
-v1.8.x will be the last versions to support 2.7 and 3.3.
+``getconf`` v1.10 onwards supports Python 3.5, 3.6, 3.7 and 3.8 and is distributed under the two-clause BSD license.
+v1.9.x are the last versions to support 2.7 and 3.4.
+v1.8.x are the last versions to support 3.3.
 The latest version of ``getconf`` to support Python 2.6 is v1.5.1.
```

Comment:
Is there a reason for the different wording of those "last version to support" sentences? ;)

Reply:
I'll ask the person who wrote them first. ;) But yes, you're right, I will reformulate the last sentence.